crates/languages/src/css/highlights.scm

```diff
@@ -25,20 +19,29 @@
   "only"
 ] @operator
 
+[
+  (tag_name)
+  (nesting_selector)
+  (universal_selector)
+] @tag
+
+(universal_selector ("*") @tag)
+
 (attribute_selector (plain_value) @string)
 
 (attribute_name) @attribute
-(pseudo_element_selector (tag_name) @attribute)
-(pseudo_class_selector (class_name) @attribute)
 
 [
   (class_name)
   (id_name)
   (namespace_name)
   (feature_name)
```

Comment:
Same with `feature_name`, we can probably do better than `@attribute` here, attribute doesn't really fit.